x-pack/plugins/profiling/public/components/flamegraph.tsx

```diff
@@ -70,7 +66,7 @@ export const FlameGraph: React.FC<FlameGraphProps> = ({ id, height }) => {
             valueAccessor={(d: Datum) => d.value as number}
             valueFormatter={(value) => `${value}`}
             animation={{ duration: 250 }}
-            controlProviderCallback={() => {}}
+            controlProviderCallback={{}}
```

Comment:
the component actually expects an object of functions, not a function by itself.